src/ensembl/src/shared/components/in-app-search/InAppSearch.tsx

```diff
@@ -82,11 +82,14 @@ const InAppSearch = (props: Props) => {
     dispatch(clearSearch({ app, genomeId }));
   };
 
+  const questionButtonHint =
+    'Find a gene using a stable ID (versioned or un-versioned), symbol or synonym; wildcards are also supported';
+
   const rightCorner = query ? (
     <CloseButton onClick={clear} />
   ) : (
     <QuestionButton
-      helpText="This is a hint"
+      helpText={questionButtonHint}
```

Comment:
We recently agreed to pass string values for attributes as string literals values rather than expressions. Do we make an exception since it is too long?

Reply:
I would guess so. It looked a bit odd to put it all in a single line.